src/libraries/System.Private.Xml/src/System/Xml/Core/XmlParserContext.cs

```diff
@@ -34,17 +35,17 @@ public XmlParserContext(XmlNameTable nt, XmlNamespaceManager nsMgr, string xmlLa
             // Intentionally Empty
         }
 
-        public XmlParserContext(XmlNameTable nt, XmlNamespaceManager nsMgr, string docTypeName,
-                  string pubId, string sysId, string internalSubset, string baseURI,
+        public XmlParserContext(XmlNameTable nt, XmlNamespaceManager nsMgr, string? docTypeName,
+                  string? pubId, string? sysId, string? internalSubset, string baseURI,
```

Comment:
Plus to `XmlNameTable nt, XmlNamespaceManager nsMgr, string xmlLang`  also `string baseURI` can be nullable, might be not the intention though